presto-main/src/main/java/com/facebook/presto/server/TaskResource.java

```diff
@@ -298,65 +275,6 @@ public TaskInfo deleteTask(
         return taskInfo;
     }
 
-    @GET
-    @Path("{taskId}/results/{bufferId}/{token}")
-    @Produces(PRESTO_PAGES)
-    public void getResults(
```

Comment:
@arhimondr 
Why is this going way?

Reply:
It's an old implementation based on blocking IO. [AsyncPageTransportServlet.java](https://github.com/prestodb/presto/pull/21772/files#diff-a338733a9579ecb50bd4a1f6d508bb66ad7c3e8888c7165feda720cdc60095c9) implements in in a non-blocking way. Semantic wise both are identical.